js/reportFilters.js

```diff
@@ -725,7 +725,8 @@ jQuery(document).ready(function ($) {
         return r.join('<br/>');
       },
       loadForm: function (context) {
-        if (context && context.quality && context.quality !== 'all') {
+        //if (context && context.quality && context.quality !== 'all') {
+        if ((context && context.quality && context.quality !== 'all') || ($('.status-filters').length && $('.status-filters').val())) {
```

Comment:
The idea behind "context" is you can have filters which define a selection of records you are permitted to use for a purpose, as decided by an administrator. Filters which you define yourself then create subsets of the former. The context is the currently selected set of records you are permitted to see, e.g. your verification filter. So, the code you've highlighted is designed on the basis that if your context specifies you are only allowed to see records of a particular status, you cannot override the status in your personal filters.

In this case though, I think the statusFilters control is not acting as a context, but really is just a shortcut to the options in the filter builder. So instead of this I think it needs a small JS script that keeps the 2 locked together. It might be worth checking with Martin and David whether it should in fact contain exactly the same options to make it less confusing?

Reply:
Thanks John. I will check with Martin and David re the list.

> I think it needs a small JS script that keeps the 2 locked together

Do you mean to implement the disabling as I have at the moment or something more sophisticated - e.g. reflecting each other's values? I tried implementing the javascript to disable the standard params quality dropdown elsewhere, but it was always them overridden by this loadForm method.

Reply:
Yes, I mean to leave it enabled, but to lock the values selected together so updating one changes the other.

Reply:
Where would be the correct place to implement the JS for that?

Reply:
To be consistent, it would be a jQuery plugin under media/js/indicia.datacomponents. You could copy one (e.g. templatedOutput) to get the code structure.